website/siteConfig.js

```diff
@@ -1,39 +1,14 @@
-/* eslint-disable sort-keys */
+/*eslint sort-keys: 0*/
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
```

Comment:
License headers need to be adjusted to MIT

Reply:
I can't adjust any additional licenses at this time.

Reply:
Still bsd here though